src/helpers/formatters/file.js

```diff
@@ -1,17 +1,20 @@
 import fs from 'fs'
+import musicMetadata from 'music-metadata'
 
-export const tags = (tags, file) => {
+export function tags (tags, file) {
```

Comment:
@Borewit Please leave it as `export const tags = (tags, file) => {`.
It's also local convention.

Reply:
It's time to break cut that convention loose.

Reply:
Are worried about scope pollution using the `function`?